docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,224 @@
+
+## Setting up a MASTER_SAFE
+
+In the upcoming sections, all scripts are executed on Rinkeby only.
+In order to perform the steps on Mainnet, you need to do some obvious modifications. For example, you would want to go to [https://gnosis-safe.io/](https://gnosis-safe.io/) instead of [https://rinkeby.gnosis-safe.io/](https://rinkeby.gnosis-safe.io/).
+
+First, we will generate a Gnosis-Safe account. This account is needed to bundle a lot of Ethereum transactions to 2 only transactions to simplify the whole setup process and to safe gas.
+
+1. Visit: https://rinkeby.gnosis-safe.io/ and sign into MetaMask with your "signer account".
+
+![Gnosis-Interface landing page](assets/tutorial_bracket_strategy-safe-start-page.png)
+
+2. Create a new Gnosis-Safe by following the steps on the display:
+
+- give your Gnosis-Safe a name: `MASTER_SAFE`
+- add a second owner, the "proposer account"
+- set the threshold for executing transactions to 2
+  ![Owners and confirmation](assets/tutorial_bracket_strategy_safe-setup.png)
+
+3. Finish the setup process by signing the MetaMask transaction
+
+4. Now, you should see your brand new gnosis-safe. Congrats. Send the funds you want to invest into the bracket strategy to the Gnosis-Safe.
+
+## Liquidity provision
+
+Get the code of github repo: https://github.com/gnosis/dex-liquidity-provision by running:
+
+```ssh
+cd <your preferred dictionary>
+git clone git@github.com:gnosis/dex-liquidity-provision.git
+cd dex-liquidity-provision
+yarn install
+npx truffle compile
+yarn run networks-inject
+```
+
+`yarn install` will install all dependencies needed and `npx truffle compile` will compile your the necessary contracts.
+`yarn run networks-inject` will make sure that you will work with the correct contracts - it will inject the network addresses into your truffle builds.
+
+You find the core script placing orders for the basic liquidity strategy in the folder `scripts/complete_liquidity_provision.js`.
+
+Before you start this script, you need to export some environment variables and prepare the parameters.
+
+### Env variables
+
+Run the following commands in your console:
+
+```ssh
+export NETWORK_NAME=rinkeby
+export GAS_PRICE_GWEI=<gas price>
+export MASTER_SAFE=<your safe address>
+export PK=<your private key of the proposer-account>
+```
+
+The `NETWORK_NAME` should be set to Mainnet or Rinkeby. The gas price should be set in a way that your transaction gets mined in a reasonable time. Check out reasonable gas prices on [ethgasstation](https://ethgasstation.info/).
+The `MASTER_SAFE` should be the safe address from the Gnosis-Safe created in the previous section.
+The `PK` should be the private key of your proposer account used as an owner in the Gnosis-Safe.
+Note, that in our current setup the PK is not secured with the highest standards.
```

Comment:
```suggestion
Note that in our current setup the proposer account's private key is not handled safely, since it's available as an environment variable and in the command-line history.
```